src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printList(Task[] list, int counter){
+        for(int i = 0; i < counter; i++){
+            System.out.print(i + 1 + ".");
+            printTask(list[i]);
+        }
+    }
+    public static void printTask(Task t){
+       System.out.println("[" + t.getStatusIcon()+ "] " + t.taskName);
+    }
+
+    public static int getTaskNumberArgument(String input){
+        int spaceIndex = input.indexOf(" ");
+        String taskNum = input.substring(spaceIndex + 1);
+        System.out.println(taskNum);
+        return Integer.parseInt(taskNum.trim());
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+        Task[] list=new Task[100];
```

Comment:
Maybe could use a different name for the task list as just "list" is vague in what it is storing

Reply:
Got it!